test/Microsoft.Framework.PackageManager.FunctionalTests/PackageManagerFunctionalTestFixture.cs

```diff
@@ -19,10 +19,28 @@ public PackageManagerFunctionalTestFixture() : base()
             Directory.CreateDirectory(PackageSource);
 
             CreateNewPackage("alpha", "0.1.0");
+
+            PackPackage(Path.Combine(TestUtils.GetMiscProjectsFolder(), "XreTestApps/CommandsProject"), PackageSource);
```

Comment:
Um... is this running for every test class in the functional tests folder? Isn't it supposed to run only for your test?